README.md

```diff
@@ -0,0 +1,153 @@
+<a name="readme-top"></a>
+
+<div>
+
+  <h1><b>React Capstone</b></h1><br/><br/>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 React Capstone Project<a name="about-project"></a>
+
+**Ezzy Phone Specs**
+[Description of project](https://www.loom.com/share/671e21bc8fe44f47a2b367d0d1dbddba)
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<summary>Client</summary>
+<ul>
+  <li><a href="https://reactjs.org/">ReactJS</a></li>
+  <li><a href="https://reactrouter.com/en/main">React Router</a></li>
+  <li><a href="https://redux.js.org/redux-toolkit/overview">Reduc Toolkit</a></li>
+</ul>
+
+### Key Features <a name="key-features"></a>
+
+- **Display list of brands to select from on app load**
+- **Display list of phones to select from when a brand is selected**
+- **Display list of phone attributes and phone specification button to show you
+  the specs of the phone**
+- **Display in details the specs of the phone**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+[Live Demo](https://react-capstone-nine.vercel.app/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- A browser (e.g. Firefox)
+- An IDE (e.g. [Visusal Studio Code](https://code.visualstudio.com/download))
+- [npm](https://nodejs.org/en/)
+- Terminal
+
+### Setup
+
+To clone this repository to your desired folder, follow the steps below:
+
+**Using the command line**
+
+- Use the following commands
+
+```sh
+  cd my-folder
+  git clone https://github.com/Ezzywealth/React-Capstone
+```
+
+**Using GitHub Desktop app**
+
+- Follow the steps below
+
+  - Visit this link "https://github.com/Ezzywealth/React-Capstone"
+  - Click the green button labelled "code"
+  - Select the "Open with GitHub Desktop" option
+  - After the GitHub Desktop add opens, click the "clone repo" button
+
+### Usage
+
+To run the project, execute the following command: **npm install** **npm run
+dev**
```

Comment:
what the previous reviewer was trying to hint you on is that your readme lacks a section called run tests and it is in this section you'd include the command for running tests. Kindly include this command to make the readme more detailed.